protocols/cmp/presign/abort1.go

```diff
@@ -22,21 +22,29 @@ type abort1 struct {
 	DeltaAlphas map[party.ID]map[party.ID]*safenum.Int
 }
 
-type messageAbort1 struct {
+type broadcastAbort1 struct {
 	// GammaShare = γᵢ
 	GammaShare  *safenum.Int
 	KProof      *abortNth
 	DeltaProofs map[party.ID]*abortNth
 }
 
-// VerifyMessage implements round.Round.
-func (r *abort1) VerifyMessage(msg round.Message) error {
+// StoreBroadcastMessage implements round.BroadcastRound.
+func (r *abort1) StoreBroadcastMessage(msg round.Message) error {
 	from := msg.From
-	body, ok := msg.Content.(*messageAbort1)
+	body, ok := msg.Content.(*broadcastAbort1)
 	if !ok || body == nil {
 		return round.ErrInvalidContent
 	}
 
+	alphas := make(map[party.ID]*safenum.Int, len(body.DeltaProofs))
```

Comment:
We need to store these before actually verifying, since they may be required to attribute blame in `Finalize`.